### DISCUSSION TITLE
Add Koa, others to benchmarks

### DISCUSSION THREAD

**Discussion by shellscape** [adonisjs/core#4439] [Ideas]
I'd like to see Koa and others added to benchmarks. I think it'd be an interesting comparison.

**Comment by RomainLanz** [comment]
Hey @shellscape! 👋🏻 

There's no need to add all the Node.js frameworks in our benchmark test suite.
If you are interested in those "dumb" benchmark, feel free to checkout [Web Frameworks Benchmark](https://web-frameworks-benchmark.netlify.app/result?f=adonisjs6-http,koa)

**Reply by shellscape** [reply]
did not know that existed! would be a handy link from your benchmarks readme.

**Reply by thetutlage** [reply]
Hello world benchmarks are not something we celebrate or push. Simply because they are far from the reality or how actual apps are written.

**Reply by McSneaky** [reply]
[Techempower benchmarks](https://www.techempower.com/benchmarks/#hw=ph&test=fortune&section=data-r22) are slightly more useful, since they also have DB calls, inserts, template renders etc in there. Slighty more "real world" than "Hello World", but still real far from actual applications
